server/app.py

```diff
@@ -0,0 +1,17 @@
+# server.py
+from flask import Flask, render_template, request, url_for
+import speak
+
+app = Flask(__name__, static_folder="../static/dist", template_folder="../static")
+@app.route("/", methods=["GET","POST"])
+def index():
+    return render_template("index.html")
+
+@app.route("/hello/<text>/<lang>")
+def hello(text,lang):
+    if request.method == "GET":
+       mp3_path = speak.get_speak(text,lang)
+       return url_for("static", filename="mp3_path")
```

Comment:
now `filename` is getting string `filename="mp3_path"`, it should be the variable name instead `filename=mp3_path`

Reply:
Thanks ! Changed a small error at the random string name too...

Reply:
Actually an error comes up with :

return url_for("static", "mp3_path")
or either: 
return url_for("static", mp3_path)


TypeError: url_for() takes 1 positional argument but 2 were given

Reply:
it should be `return url_for("static", filename=mp3_path)`